iac-extensions/terraform/src/test/java/org/sonar/iac/terraform/plugin/TerraformSensorTest.java

```diff
@@ -64,7 +65,8 @@ void test_one_rule() {
     IssueLocation location = issue.primaryLocation();
     assertThat(location.inputComponent()).isEqualTo(inputFile);
     assertThat(location.message()).isEqualTo("Rename tag key \"anycompany:cost-center\" to match the regular expression \"^([A-Z][A-Za-z]*:)*([A-Z][A-Za-z]*)$\".");
-    TextRangeAssert.assertTextRange(location.textRange()).hasRange(2, 11, 2, 35);
+    TextRangeAssert.assertTextRange(TextRanges.range(location.textRange().start().line(), location.textRange().start().lineOffset(), location.textRange().end().line(),
```

Comment:
As this is quite hard to read, maybe you can use a variable for `location.textRange()` and for the results of `TextRanges.range()`